web-wallet/src/lib/url/__tests__/makeNodeUrl.spec.js

```diff
@@ -0,0 +1,155 @@
+import { describe, expect, it } from "vitest";
+import { makeNodeUrl } from "..";
+// import { JSDOM } from "jsdom";
+
+describe("makeNodeUrl", () => {
+  const localhostString = window.location.hostname;
+
+  it("should return a local URL when VITE_NODE_URL is not set", () => {
+    delete import.meta.env.VITE_NODE_URL;
```

Comment:
It's not a good idea to delete things, as this can affect subsequent tests.

Vitest has its helpers anyway to deal with env variables: [`vi.stubEnv`](https://vitest.dev/api/vi.html#vi-stubenv) and [`vi.unstubAllEnvs`](https://vitest.dev/api/vi.html#vi-unstuballenvs).

Reply:
Ah ok, I wasn't aware of that API.  I will update the tests.